bin/test.bash

```diff
@@ -23,17 +25,58 @@ test_apps_wc() {
         "${pipeline_path}/test/services/test-wc.sh" "${config[config_file_wc]}"
 }
 
-#
-# ENTRYPOINT
-#
-
-config_load "$1"
-if [[ $1 == "wc" ]]; then
-    test_apps_wc
-elif [[ $1 == "sc" ]]; then
-    test_apps_sc
-else
-    echo "ERROR:  [$1] is an invalid argument:"
-    echo "usage:   ck8s test <wc|sc>. "
-    exit 1
-fi
+function sc_help(){
+    printf "%s\n" "[Usage]: test sc [target] [ARGUMENTS]";
+    printf "%s\n" "List of targets:";
+    printf "\t%-23s %s\n" "opensearch" "Open search checks";
+    printf "\t%-23s %s\n" "fluentd" "Fluentd checks";
+    printf "\t%-23s %s\n" "opensearch-dashboards" "Opensearch dashboards checks";
+    printf "%s\n" "[NOTE] If no target is specified, the tests will go through all of them.";
+    exit 0;
+}
+
+function sc() {
+    if [[ ${#} == 0 ]]; then
+        test_apps_sc
+    else
+        case ${1} in
+            opensearch )
+                sc_opensearch_checks "${@:2}";
```

Comment:
```suggestion
                with_kubeconfig "${config[kube_config_sc]}" \
                    sc_opensearch_checks "${@:2}";
```
Else the kubeconfig won't be set properly. I think we need to rethink that in general here since it doesn't make much sense how it's done a bit of everywhere, but we can do it in another issue when we refactor this a bit.

Reply:
I'm not sure how it is not set properly, but in my case, It's working fine. 
I've modified in the main function so it looks like this : 
```

function main(){
    if [[ ${#} == 0 ]]; then
        main_help 0;
    fi

    case ${1} in
        sc )
            config_load "$1";
            with_kubeconfig "${config[kube_config_sc]}" \
              "$1" "${@:2}";
        ;;
        wc )
            config_load "$1";
            with_kubeconfig "${config[kube_config_wc]}" \
              "$1" "${@:2}";
        ;;
    esac
}
```